tests/test_single_topology_v3.py

```diff
@@ -439,7 +439,7 @@ def test_interpolate_harmonic_force_constant(src_k, dst_k, k_min, lambda_interva
     assert f(src_k, dst_k, 0.0) == src_k
     assert f(src_k, dst_k, 1.0) == dst_k
 
-    lambdas = np.linspace(lambda_min, lambda_max, 10)
+    lambdas = np.arange(0.01, 1.0, 0.01)
```

Comment:
If I understand the `given` code, lambdas[0] can be greater than lambda_min, is that true and expected?

Reply:
Yes, that is true and expected -- the interpolation function should be nondecreasing over the whole interval (excluding the end points) regardless of lambda_{min,max}